src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java

```diff
@@ -63,6 +63,9 @@ public void testGenerateFeedbackSessionEmails() throws IOException {
 
         StudentAttributes student1 = studentsLogic.getStudentForEmail(course.getId(), "student1InCourse1@gmail.tmt");
 
+        // Unregistered student
+        StudentAttributes student2 = studentsLogic.getUnregisteredStudentsForCourse("idOfUnregisteredCourse").get(0);
```

Comment:
We can avoid the comment and rename the variable to `unregisteredStudent`. Also how about using `getStudentForEmail("idOfUnregisteredCourse", "student1InUnregisteredCourse@gmail.tmt");`